.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,85 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      production_host:
+        required: true
+        type: string
+        description: 'The pruduction server'
+      playbook_path:
+        required: true
+        type: string
+        description: 'The path to the file relative to the repository'
+      deploy_path:
+        required: true
+        type: string
+        description: 'The path to deploy on the production server'
+      deploy_repo:
+        required: true
+        type: string
+        description: 'The repository for deploy'
+      deploy_branch:
+        required: true
+        type: string
+        description: 'The branche for deploy'
+      python_version:
+        required: true
+        type: number
+        description: 'The version of python'
+      runner_image:
+        required: true
+        type: string
+        description: 'The image of the runner OS on which workflow is running'
+    # Secrets for ssh
+    secrets:
+      ssh_deploy_user:
+        required: true
+        description: 'The username for connect to production server via ssh'
+      ssh_private_key:
+        required: true
+        description: 'The private key for connect to production server via ssh'
+
+jobs:
+  deploy:
+    runs-on: ${{ inputs.runner_image }}
+
+    steps:
+      # Preparing workspace: install python, ansible and ansistrano-role
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ inputs.PYTHON_VERSION }}
+      - name: Install Ansible
+        run: |
+          python -m pip install --upgrade pip
+          pip install ansible requests jinja2
+      - name: Install Ansistrano roles
+        run: |
+          ansible-galaxy install ansistrano.deploy ansistrano.rollback
+
+      # Adding private-key to ssh-agent
+      - name: Set up ssh-agent
+        uses: webfactory/ssh-agent@v0.5.2
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+      # Deploy the project
+      - name: Ansistrano deploy
+        env:
+          # Environment variables for ansible must be in uppercase
+          ANSIBLE_HOST_KEY_CHECKING: False
```

Comment:
why False?

Reply:
To exclude the occurrence of an interactive query when executing a playbook

Reply:
You can add host to known_hosts on the previous step to avoid this, can't you?

Reply:
done